ui/src/app/views/registration-dashboard.tsx

```diff
@@ -195,24 +208,71 @@ export class RegistrationDashboard extends React.Component<RegistrationDashboard
     }
   }
 
+  async setAllModulesBypassed(isBypassed: boolean) {
+    this.setState({bypassInProgress: true});
+
+    // TypeScript enum iteration is nonfunctional
+    // so just copy the whole list
+    const modules = [
+      AccessModule.COMPLIANCETRAINING,
+      AccessModule.ERACOMMONS,
+      AccessModule.TWOFACTORAUTH,
+      AccessModule.DATAUSEAGREEMENT,
+      AccessModule.BETAACCESS
+    ];
+
+    for (const module of modules) {
```

Comment:
Side note: I learned that in order to run a series of promises in serial (rather than parallel), you need to use a for-loop and 'await' calls rather than array.forEach. I'll do a quick follow-up change to make our user admin UI do the same thing when it runs its bypass loop (https://github.com/all-of-us/workbench/blob/master/ui/src/app/views/admin-user-bypass.tsx#L57)

Reply:
👍